assets/src/edit-story/components/library/panes/media/media3p/media3pCategories.js

```diff
@@ -56,26 +58,44 @@ const Media3pCategories = ({
   deselectCategory,
 }) => {
   const [isExpanded, setIsExpanded] = useState(false);
+
+  function renderCategories() {
+    return categories.map((e) => {
+      return (
+        <CategoryPill
+          isSelected={false}
+          key={e.id}
+          title={e.displayName}
+          onClick={() => {
+            setIsExpanded(false);
+            selectCategory(e.id);
+          }}
+        />
+      );
+    });
+  }
+
+  function renderSelectedCategory() {
+    const selected = categories.find((e) => e.id === selectedCategoryId);
+    return (
+      <CategoryPill
+        isSelected={true}
+        key={selected.id}
+        title={selected.displayName}
+        onClick={() => deselectCategory()}
+      />
+    );
+  }
+
   return categories.length ? (
     <CategorySection aria-expanded={isExpanded}>
       <CategoryPillContainer isExpanded={isExpanded} role="tablist">
-        {categories.map((e) => {
-          const selected = e.id === selectedCategoryId;
-          return (
-            <CategoryPill
-              isSelected={selected}
-              key={e.id}
-              title={e.displayName}
-              onClick={() =>
-                selected ? deselectCategory() : selectCategory(e.id)
-              }
-            />
-          );
-        })}
+        {selectedCategoryId ? renderSelectedCategory() : renderCategories()}
       </CategoryPillContainer>
       <ExpandButton
         onClick={() => setIsExpanded(!isExpanded)}
         isExpanded={isExpanded}
+        hidden={selectedCategoryId}
```

Comment:
It might be better to have a `visible` param instead, to avoid using the negative.

Reply:
Done.